src/components/UI/MyFriendsCard.jsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Card, CardContent, CardMedia, styled } from '@mui/material'
+import React from 'react'
+
+import CardImage from '../../assets/images/Card-Image.png'
```

Comment:
import is not correct you should to import React Component
import CardImage from '../../assets' - it's should work